### DISCUSSION TITLE
Inputs & Dropdown hover text resets

### DISCUSSION THREAD

**Discussion by megaroeny** [primefaces/community#1269] [Figma UI Kit]
I can't figure out how to fix this, but one very annoying issue for all Form input/dropdown fields is with the hover and active states. The input text (placeholder and text) always resets and makes using them in prototypes very annoying. I have tried going through all the props and still can't understand why. This doesn't usually happen for Input fields, but it does for Dropdowns. Probably because there's too much nesting happening, and something is breaking because of it.

**See this example:**

https://github.com/primefaces/community/assets/1559447/ff09195b-2157-43f0-b535-cf2246037fba

## Suggested solution

I think one reason might be because you are relying on the nested Input instance states for hover, but the Dropdown main component does not have those variants/states. So it's not respecting the nested text overrides for the Inputs. If you made dedicated `hover` state variants for Dropdown, I think it might fix this issue.

With that said, that is a lot of maintenance and extra variants you could probably optimize for. Something I'd suggest is create a component just for the input "box" with it's various states. Then that gets nested in all your Form fields (similar to what you did for `_input-content`)

Thanks for the help!

**Comment by onursenture** [comment]
Hello Reony, 

I have now tested dropdown and multiselect by drag&dropping them from the local components to the playground and then changing their text values; I encountered no problems in the prototype. 

https://github.com/primefaces/community/assets/244957/d383bb5f-bea7-4714-ae32-04a5706127fa

https://github.com/primefaces/community/assets/244957/2b0c8e56-c37d-4643-80fb-49c76ec559c7

Could you please confirm if the problem occurred after some modifications you made? You could test with a PrimeOne file in which no changes were made to verify this. And if the problem still persists in this situation, could you guide us on the steps we should take to reproduce this issue? 

From what I can see in the screencast, the first appearance of 'input_content' appears to be in the Focus state, which isn't typically the case under normal conditions.

**Reply by megaroeny** [reply]
Correct. If we want to show a dropdown menu with a value selected instead of a placeholder, I switch it to "Text" option. That's when I see this issue happening.

**Reply by onursenture** [reply]
Ah, yes, now I can also reproduce the problem. And yes, the issue is resolved when I add a hover state for the dropdown. I'm noting that for the next release, we'll likely make this change in all components that use the input. Thank you so much! 🙌

**Reply by megaroeny** [reply]
Fantastic! Glad to help.
